src/extension.ts

```diff
@@ -110,6 +112,26 @@ export class Extension implements vscode.Disposable {
       ? Promise.resolve()
       : this.setEditorMode(editor, mode)
   }
+  
+  setLineNumberStyle(editor: vscode.TextEditor, style:string) {
```

Comment:
```suggestion
  private setLineNumbersStyle(editor: vscode.TextEditor, style: string) {
```

(also requires change above)